source/stylesheets/_footer.scss

```diff
@@ -1,43 +1,47 @@
 footer {
   font-size: 18px;
   font-family: opensans-regular;
-}
-
-footer h4 {
-  font-family: opensans-semibold;
-  font-size: 16px;
-  padding-bottom: 12px;
-}
 
-footer .social-icon {
-  font-size: 26px;
-  color: $dp-light-gray;
-  padding-right: 30px;
+  h4 {
+    font-family: opensans-semibold;
+    font-size: 16px;
+    padding-bottom: 0px;
+  }
+
+   .social-icon {
+    font-size: 26px;
+    color: $dp-light-gray;
+    padding-right: 30px;
+  }
+
+  .col-lg {
+    padding-bottom: 20px;
+  }
+
+  p {
+    padding-top: 12px;
+  }
 }
 
 #mc_embed_signup {
-  margin-top: -5px;
-}
-
-footer .col-lg {
-  padding-bottom: 20px;
+  margin-top: 8px;
 }
 
 @media (max-width: 991px) {
```

Comment:
I do not love this, I'd like to use bootstrap mixins like `media-breakpoint-up` but I don't think we can import bootstrap if we're have it as an already compiled CSS. @avand what do you think? Is it worth it to import bootstrap saas, or leave it as it is right now?